### ISSUE TITLE
Add Support for Button Switches

### ISSUE THREAD

**Issue by jollypotato246** [bwp91/homebridge-govee#735]
{"device":"AB:96:D8:32:39:37:65:67","deviceName":"Button Switch 1","model":"H5126","httpInfo":{"groupId":0,"device":"AB:96:D8:32:39:37:65:67","sku":"H5126","spec":"","versionHard":"1","versionSoft":"1","deviceName":"Button Switch 1","deviceExt":{"deviceSettings":"{\"address\":\"D8:32:39:37:65:67\",\"bleName\":\"GV51266567\",\"ic\":0,\"sku\":\"H5126\",\"device\":\"AB:96:D8:32:39:37:65:67\",\"deviceName\":\"Button Switch 1\",\"versionHard\":\"1\",\"versionSoft\":\"1\",\"battery\":80,\"time\":1707711149889}","lastDeviceData":"{\"online\":true,\"bind\":true}","extResources":"{\"skuUrl\":\"https://d1f2504ijhdyjw.cloudfront.net/sku-img/4c945b48a243de461fe6572217878c10-add_list_type_device_5126.png\",\"headOnImg\":\"\",\"headOnImgNew\":\"\",\"headOffImg\":\"\",\"headOffImgNew\":\"\",\"ext\":\"\",\"ic\":0}"},"share":0,"deviceId":16339176,"goodsType":145,"attributesId":0,"supportScene":0}}

**Comment by mikedsahgdotnet** [comment]
Yes please! I agree. They now have a lot of different buttons. I just purchased an H5126 myself.
